dali/operators/image/resize/resize_attr.cc

```diff
@@ -22,66 +22,85 @@
 namespace dali {
 
 DALI_SCHEMA(ResizeAttr)
-  .AddOptionalArg("resize_x", R"(The length of the X dimension of the resized image.
+  .AddOptionalArg("resize_x", R"code(The length of the X dimension of the resized image.
```

Comment:
We don't need this "code" everywhere. It was used once in one place ages ago and now it spreads like cancer through our codebase :( :(

Reply:
We can just run seed through code base to replace that. But I would like to have an agreement regarring the replacement.